src/consultations/consultations.controller.ts

```diff
@@ -91,7 +91,11 @@ export class ConsultationsController {
   ): Promise<ConsultationDetailsDto> {
     const participation = await this.participationService.findOne(id, user.id)
     try {
-      return await this.consultationsService.findOne(id, user, participation?.id)
+      return await this.consultationsService.findOne(
+        id,
+        user,
+        participation?.id,
+      )
     } catch {
       throw new HttpException(
         'A konzultáció nem található!',
```

Comment:
Not related to this issue, but `consultationService.findOne` already handles the Not Found case, no need for the try-catch here,